scripts/protocgen-pulsar.sh

```diff
@@ -0,0 +1,22 @@
+# this script is for generating protobuf files for the new google.golang.org/protobuf API
```

Comment:
protocgen-pulsar.sh generates the pulsar proto files for depinject `Module` types